python/kserve/kserve/protocol/dataplane.py

```diff
@@ -50,23 +51,23 @@ def __init__(self, model_registry: ModelRepository):
     def model_registry(self):
         return self._model_registry
 
-    def get_model_from_registry(self, name: str) -> Union[Model, RayServeHandle]:
+    def get_model_from_registry(self, name: str) -> Union[Model, _DeploymentHandleBase]:
```

Comment:
Seems like we are not supposed to use `_DeploymentHandleBase` as underscore indicates that it is not exposed?

Reply:
it's a forward-compatible line to account for the upcoming API changes to serve.run returning a `DeploymentHandle`. I couldn't find much documentation about it, but users who enable the feature flag would break this setup if we don't use the common base class here (which alas has an underscore indeed).

see also https://github.com/ray-project/ray/pull/37817

Reply:
regarding this type hint specifically: the registry can now contain either `RayServeHandle`, `RayServeSyncHandle`, or the new `DeploymentHandle` if the user sets the env var. all three share this base class.

Reply:
fwiw, I'm not into ray at all. I just tried to get the CI green with a minimal diff. I'm out of office until end of October, feel free to take over the branch in the meantime!

Reply:
@ddelange can you help sign off the commit ?

Reply:
done 👍